apistructs/qa.go

```diff
@@ -186,7 +186,7 @@ type Test struct {
 	Classname string        `json:"classname" yaml:"classname"`
 	Duration  time.Duration `json:"duration" yaml:"duration"`
 	Status    TestStatus    `json:"status" yaml:"status"`
```

Comment:
is this apistructs useless? shall we delete it?

Reply:
apistructs are used by ut-action

Reply:
![image](https://user-images.githubusercontent.com/30427474/167755331-141ced81-b527-4d80-8085-6ab69f773d7c.png)
pipeline and some modules are uesd

Reply:
adjuste ut-action use pb, and some pkg use pb